doc/news/prym.rst

```diff
@@ -0,0 +1,3 @@
+**Added:**
+
+* Added ``translation_surfaces.lanneau_nguyen_genus3_prototype()`` and ``translation_surfaces.lanneau_nguyen_genus4_prototype()``.
```

Comment:
@sfreedman67 feel free to modify this to say something more inspiring.

Reply:
I was following the style of ``translation_surfaces.mcmullen_genus2_prototype()``

Reply:
Sorry, I meant feel free to make the "news" sound more exciting.

I saw that you tried to follow the style of the genus 2 implementation closely. Imho that implementation is more complicated than it needs to be so I tried to simplify it a bit.